rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/project/ProjectController.java

```diff
@@ -1227,4 +1228,27 @@ public void getUserProjectCount(HttpServletResponse response) throws TException
             throw new SW360Exception(e.getMessage());
         }
     }
+    @RequestMapping(value = PROJECTS_URL + "/{id}/showChildProject", method = RequestMethod.GET)
```

Comment:
From Disussion, endpint should be similar to componet endpoit https://sw360.siemens.com/resource/docs/api-guide.html#resources-component-usedby-list